apps/demos/project.json

```
@@ -0,0 +1,121 @@
+{
+  "name": "devextreme-demos",
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "apps/demos",
+  "projectType": "application",
+  "targets": {
+    "test": {
+      "executor": "nx:run-script",
+      "options": {
+        "script": "test"
+      },
+      "dependsOn": [
+        "!^build"
+      ],
```

Does nx support excluding targets from `dependsOn`? Shouldn't it be like so?
```suggestion
      "dependsOn": [],
```